change/@ni-nimble-angular-ae616687-e489-4588-ba8f-583778696c91.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Just a heads-up: for the table we decided to call `checkValidity()` in some places in our SLE `sl-table` unit tests to help catch occasions where the app has configured the Nimble component incorrectly. You could consider doing the same for the rich text components in a follow up PR.

[Example here](https://dev.azure.com/ni/DevCentral/_search?action=contents&text=checkvalidity&type=code&lp=code-Project&filters=ProjectFilters%7BDevCentral%7DRepositoryFilters%7BSkyline%7D&pageSize=25&result=DefaultCollection/DevCentral/Skyline/GBmaster//Web/Workspaces/SystemLinkShared/projects/systemlink-lib-angular/src/sl-table/sl-table.component.spec.ts)

Reply:
Thanks for the heads-up. I have created a TD to track the work of writing pageobjects in `sl-comments`: [Technical Debt 2628236](https://dev.azure.com/ni/DevCentral/_workitems/edit/2628236): FE | Add `checkValidity()` for sl-comments unit tests